actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -177,7 +177,7 @@ private void cacheFrameworkFieldsConfig() {
   private void updateUser(Request actorMessage) {
     Util.initializeContext(actorMessage, TelemetryEnvKey.USER);
     actorMessage.toLower();
-    Util.getUserProfileConfig(systemSettingActorRef);
+    //Util.getUserProfileConfig(systemSettingActorRef);
```

Comment:
sure?

Reply:
commented as this line of code just reading the data from system setting table but not using that